tests/manager/manager-ctor-duplicate-option-001.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+MongoDB\Driver\Manager::__construct() with duplicate read preference option
+--FILE--
+<?php
+
+$manager = new MongoDB\Driver\Manager(null, ['readPreference' => 'primary', 'readpreference' => 'secondary']);
```

Comment:
That's a good point. I'd assume that overwriting is indeed the behaviour we're looking for, as opposed to merging tag sets, correct?

Reply:
Test added in squashed commit.

Reply:
Correct. Merging tag sets is done for the connection string because we otherwise have no way to build an array (similar to how most frameworks interpret duplicate `key[]` form fields). But that doesn't make sense for the array options, since an entire tag set can easily be expressed in a single key. Therefore, overwriting makes sense in all cases for array options.

Reply:
That sounds good. Thanks for confirming!